libopae/CMakeLists.txt

```diff
@@ -43,6 +43,8 @@ target_include_directories(opae-c PUBLIC
 
 include_directories(${libjson-c_INCLUDE_DIRS})
 
+include_directories(${libjson-c_INCLUDE_DIRS})
```

Comment:
Is this line a duplicate of the above line?

Reply:
I think so. Thanks, @lzhan55